src/tideliftMeUp.ts

```diff
@@ -37,9 +38,22 @@ export async function tideliftMeUp({
 	const allUserPackages = await npmUsernameToPackages(username);
 
 	if (!allUserPackages.length) {
-		throw new TideliftMeUpError(
-			`No packages found for npm username: ${username}.`,
-		);
+		try {
+			await npmUser(username);
+			throw new TideliftMeUpError(
+				`No packages found for npm username: ${username}.`,
+			);
+		} catch (error) {
+			if (
+				typeof error === "object" &&
+				error !== null &&
+				"code" in error &&
+				error.code === "ERR_NO_NPM_USER"
+			) {
+				throw new TideliftMeUpError(`Invalid npm username.`);
```

Comment:
[Feature] Could this also print the username? If I'm sending in invalid data I might not be sending the username I thought.

```suggestion
				throw new TideliftMeUpError(`Invalid npm username: ${username}.`);
```